mlir/include/mlir/IR/AffineMap.h

```diff
@@ -595,7 +595,10 @@ AffineMap inverseAndBroadcastProjectedPermutation(AffineMap map);
 /// potentially empty maps. Assumes each of the underlying map has 0 symbols.
 /// The resulting map has a number of dims equal to the max of `maps`' dims and
 /// the concatenated results as its results.
-/// Returns an empty map if all input `maps` are empty.
+///
+/// This method asserts when `maps` is empty.
+/// TODO: this should return an empty map when `maps` is empty but there is no
+/// way to get the MLIRContext needed to construct it.
```

Comment:
Can we add `MLIRContext` as an argument?

Reply:
That makes sense to me. Is there a process for making a change to a public interface?

Reply:
For small changes like this (especially when it's fixing a bug), just a note in the PR description explaining how to adapt downstream use cases is good enough. Something like
```
concatAffineMaps now takes an MLIRContext in case the list of maps is empty.
```

Reply:
updated!